helper/testhelpers/docker/testhelpers.go

```diff
@@ -331,3 +333,235 @@ func copyToContainer(ctx context.Context, dapi *client.Client, containerID, from
 
 	return nil
 }
+
+type RunCmdOpt interface {
+	Apply(cfg *types.ExecConfig) error
+}
+
+type RunCmdUser string
+
+var _ RunCmdOpt = (*RunCmdUser)(nil)
+
+func (u RunCmdUser) Apply(cfg *types.ExecConfig) error {
+	cfg.User = string(u)
+	return nil
+}
+
+func (d *Runner) RunCmdWithOutput(ctx context.Context, container string, cmd []string, opts ...RunCmdOpt) ([]byte, int, error) {
```

Comment:
Does it make sense to rip out this `ctx` parameter and always make it `context.Background()`? For the tests, I'm not sure we really have much reason to do anything else, tbh.